src/main/resources/templates/search/byge/search.html

```diff
@@ -18,4 +18,13 @@
   <div th:replace="fragments/bootstrap_scripts.html"></div>
 </body>
 
+<body>
+
+
+<p><b>COE: Students only need A,D,E,F,G and H area.</b></p>
+<p><b>L&S: Students only need A,B,C,D,E,F and G area.</b></p>
+<p><b>COE: Students only need A,B,C,D,E,F and G area.</b></p>
+
+</body>
```

Comment:
Do not create another body tag. The contents should be put in the appropriate spot in the existing body tag.